nethermind/quickstart.sh

```diff
@@ -0,0 +1,652 @@
+    elif [[$DISTRIBUTION_NAME == "Fedora" ]]; then
+
+        $PERMISSION_PREFIX dnf -y install dnf-plugins-core
+
+        $PERMISSION_PREFIX dnf config-manager \
+            --add-repo \
+            https://download.docker.com/linux/${DISTRIBUTION_ID}/docker-ce.repo
+
+        $PERMISSION_PREFIX dnf install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+        echo -e "\nDocker and Docker Compose were installed for ${DISTRIBUTION_NAME}."
+    else
+        echo -e "\nWe'd support next distributions: Ubuntu, Debian, CentOS, Red Hat Enterprise Linux, Fedora. Please check out your distribution and install Docker."
+        exit -1
+    fi
+}
+
+# Function to install and configure NTP for specific Linux distribution
+function install_ntp() {
+    echo -e "\nInstall and configure NTP (chrony)...\n"
+
+    if [[ $DISTRIBUTION_NAME == "Ubuntu" || $DISTRIBUTION_NAME == "Debian" ]]; then
+        $PERMISSION_PREFIX apt-get install chrony -y
+    elif [[ $DISTRIBUTION_NAME == *"CentOS"* || $DISTRIBUTION_NAME == "Red Hat Enterprise Linux" ]]; then
+        $PERMISSION_PREFIX yum install chrony -y
+    elif [[ $DISTRIBUTION_NAME == "Fedora" ]]; then
+        $PERMISSION_PREFIX dnf install chrony -y
+    else
+        echo -e "\nWe'd support next distributions: Ubuntu, Debian, Red Hat Enterprise Linux, CentOS, Fedora. Please check out your distribution and install / configure NTP."
+        exit 1
+    fi
+
+    $PERMISSION_PREFIX systemctl stop chrony
+    $PERMISSION_PREFIX chronyd -q 'server 0.europe.pool.ntp.org iburst'
+    $PERMISSION_PREFIX systemctl start chrony
+    $PERMISSION_PREFIX systemctl enable chrony
+}
+
+# Function to check disk space
+function check_disk_space() {
+
+    # Specify disk space treshold
+    REQUIRED_DISK_SPACE_GB=50
+
+    # We're using mount volume where is quickstart.sh file
+    mounted_volume=$(df --output=target quickstart.sh | tail -n1)
+    total_volume_size_mb=$(df -k --output=size "$mounted_volume" | tail -n1)
+    total_volume_size_gb=$((total_volume_size_mb / 1024 / 1024))
+
+    # Check with specified treshold
+    if [ $total_volume_size_gb -lt $REQUIRED_DISK_SPACE_GB ]; then
+        echo -e "\nCheck disk space.... ERROR - Not enoguh total drive space! you have $total_volume_size_gb GB you require at least $REQUIRED_DISK_SPACE_GB GB!"
+        exit 1
+    else
+        echo -e "\nCheck disk space.... OK!"
+    fi
+}
+
+# Function to check RAM memory space
+function check_ram_memory_space() {
+
+    # Specify RAM memory treshold
+    REQUIRED_RAM_GB=2
+
+    # Identify RAM memory amount
+    total_ram_memory_size_mb=$(free -m | grep Mem: | awk '{print $2}')
+    total_ram_memory_size_gb=$((total_ram_memory_size_mb / 1024))
+
+    # Check with specified treshold
+    if [ $total_ram_memory_size_gb -lt $REQUIRED_RAM_GB ]; then
+        echo -e "\nCheck RAM memory space... ERROR - Not enoguh total RAM memory space! you have $total_volume_size_gb GB you require at least $REQUIRED_RAM_GB GB!"
```

Comment:
Since we don't exit here might want to print this RED and also print after the quickstart has run. (can probably copy displayWarning function from old quickstart)